mkt/search/views.py

```diff
@@ -101,6 +102,20 @@ class NonPublicSearchView(SearchView):
                        ProfileFilter, SortingFilter]
 
 
+class NoRegionSearchView(SearchView):
+    """
+    A search view that allows searching for public apps regardless of region
+    exclusions, protected behind a permission class.
+
+    """
+    authentication_classes = [RestSharedSecretAuthentication,
+                              RestOAuthAuthentication]
+    permission_classes = [GroupPermission('Feed', 'Curate')]
+    filter_backends = [SearchQueryFilter, PublicSearchFormFilter,
+                       PublicAppsFilter, DeviceTypeFilter,
```

Comment:
So what happens is, we disable geoip lookup in v2, but we default to `restofworld`. If enabled, the `RegionFilter` picks up that region (even if it's not present in the query string - giving us compatibility with the automatic geoip lookups v1 does), and that's why we need a separate endpoint without that filter.